src/main/java/view/InputView.java

```diff
@@ -1,13 +1,11 @@
 package view;
 
+import static util.constant.Message.*;
```

Comment:
WildCard Import는  [자바 코드 컨벤션](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)에 위배되는 것 같아요. 👀
```suggestion
import static util.constant.Message.INPUT_BONUS_BALL_MESSAGE;
import static util.constant.Message.INPUT_MONEY_MESSAGE;
import static util.constant.Message.INPUT_WINNING_NUMBERS_MESSAGE;
```

[IntelliJ에 이런 설정](https://www.baeldung.com/intellij-disable-wildcard-import)이 있으니 설정을 해주셔도 좋을 것 같아요. 😃